velox/common/caching/SsdFile.cpp

```diff
@@ -388,6 +393,7 @@ void SsdFile::write(std::vector<CachePin>& pins) {
     auto rc = folly::pwritev(fd_, iovecs.data(), iovecs.size(), offset);
     if (rc != bytes) {
       LOG(ERROR) << "Failed to write to SSD " << errno;
+      ++stats_.errorsWriteSsd;
```

Comment:
```
s/errorsWriteSsd/writeErrors/
```

Reply:
We have errors from writing to ssd file and to the checkpoint file. So I think we'd be better to keep 'ssd' to tell them apart.